src/content/developers/docs/web2-vs-web3/index.md

```diff
@@ -4,53 +4,54 @@ description:
-| In the event of conflicting data, resolution is clear and easy: the ultimate source of truth is the central authority.                                                                                                 | A protocol (often complex) is needed for dispute resolution, if peers make conflicting claims about the state of data which participants are meant to be synchronized on.                                                        |
-| Single point of failure: malicious actors may be able to take down the network by targeting the central authority.                                                                                                     | No single point of failure: network can still function even if a large proportion of participants are attacked/taken out.                                                                                                        |
+| Low network diameter (all participants are connected to a central authority); information propagates quickly, as propagation is handled by a central authority with lots of computational resources.                   | The furthest participants on the network may potentially be many edges away from each other. Information broadcasted from one side of the network may take a long time to reach the furthest edges.                              |
+| Usually higher performance (higher throughput, fewer total computational resources expended) and easier to implement as no consensus is required.                                                                      | Usually lower performance (lower throughput, more total computational resources expended) and more complex to implement due to complexities in achieving consensus.                                                              |
+| In the event of conflicting data, resolution is straightforward: the ultimate source of truth is the central authority.                                                                                                | A protocol (often with its own consensus mechanism and rules) is needed for dispute resolution in the event of conflicting claims about the state of data which the network should be synchronized on.                           |
+| Single point of failure: malicious actors may be able to take down the network by targeting the central authority.                                                                                                     | No single point of failure: network can still function as long as honest participants continue to form the majority of the network.                                                                                              |
```

Comment:
```suggestion
| Single point of failure: malicious actors may be able to take down the network by targeting the central authority.                                                                                                     | No single point of failure: network can still function even if a large proportion of participants are attacked/taken out.                                                                                            |
```

Theres attack vectors that occur at <50% https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/attack-and-defense/#attackers-with-33-stake so I think we should keep whats there before